clock/run.go

```diff
@@ -247,14 +246,14 @@ func isPeer(r cloudflare.Record) bool {
 }
 
 func clientFor(upstreamHost string, masqueradeHost string, rootCA string) *http.Client {
-
-	serverInfo := &client.ServerInfo{
-		Host:              upstreamHost,
-		Port:              443,
-		DialTimeoutMillis: 5000,
-	}
-	masquerade := &client.Masquerade{masqueradeHost, rootCA}
-	httpClient := client.HttpClient(serverInfo, masquerade)
-
-	return httpClient
+	d := fronted.NewDialer(&fronted.Config{
```

Comment:
@myleshorton fronted.Dialer uses a connection pool under the covers, which it's currently replenishing.  If you don't call Close at some point, this will generate a very large leak.

I'll make some tweaks to the API to support using it for one-offs like this without constructing a dialer.